apps/web/src/components/select/select.tsx

```diff
@@ -44,7 +44,7 @@ export default function Select({
   value,
   Icon,
 }: SelectProps) {
-  const _id = id ?? crypto.randomUUID()
+  const _id = id ?? '_' + Date.now().toString(36) + Math.random().toString(36).substr(2, 9)
```

Comment:
Let's switch to using `uuid` (it's already installed and ready to use). First import it at the top:

```ts
import { v4 as uuid } from 'uuid'
```

Then use it like so:

```suggestion
  const _id = id ?? uuid()
```